src/components/Loading/LoadingMessage/LoadingMessage.js

```diff
@@ -16,14 +16,16 @@ const LoadingMessage = ({
   withOverlay,
   ...other
 }) => {
-  const loading = <Loading active={active} small={small} withOverlay={false} />;
+  const loading = (
+    <Loading active={active} small={small} withOverlay={false} {...other} />
```

Comment:
`...other` needs to be moved here to pass through `Loading` component related props, like `description`.

Reply:
out of scope for this PR, but I feel a loader would need to also support aria attributes for screen readers. This spread only supports `className` and `description`

Reply:
@Deriohig The `...other` spread attribute in this case would cover everything you mentioned -- it's just passing through anything extra that's not explicitly called out. So it would pass through any number of aria attributes you gave it to the inner `div.bx--loading` element.

Reply:
Apologies, you are correct. I was using the storybooks accepted props instead of the carbon-component-react source code as my reference for the `<Loading/>` component's available props, should have dug deeper. I wonder if the support of native attributes should be documented in the storybook? Might be useful to consider.

Reply:
No worries @Deriohig I think in general the storybook needs work. 😬 I'm going to open a separate issue to address this for sure.